packages/bot-api/src/MessageHandler.ts

```diff
@@ -124,7 +124,7 @@ export abstract class MessageHandler {
   async sendCall(conversationId: string, content: CallingContent, userIds?: string[] | UserClientsMap): Promise<void> {
     if (this.account?.service) {
       const callPayload = this.account.service.conversation.messageBuilder.createCall({conversationId, content});
-      await this.account.service.conversation.send(callPayload, userIds);
+      await this.account.service.conversation.send({payloadBundle: callPayload, userIds});
```

Comment:
What makes the payload a bundle? Is it bundled with other data?

Reply:
Good question, it was already named like that.